src/shared/ui/components/checkbox/checkbox.types.ts

```diff
@@ -1,4 +1,19 @@
-export interface CheckboxProperties {
-  value: boolean;
+export type CheckboxProperties = {
+  label?: string;
+  disabled?: boolean;
+  className?: string;
+  disabledTooltipText?: string;
+  additionalClassNameWhenChecked?: string;
```

Comment:
this should be doable via className with `checked:` prefix and be injected via normal `className` property